src/pages/cityWeather/CityWeather.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import { publicApiInstance } from '../../utils/api';
+import { WeatherCard, SearchInput, SearchBar, Tooltip } from '../../components';
+import endpoints from '../../config/enpoints';
+import * as S from './CityWeather.styles';
+
+const CityWeather = () => {
+  const [cityName, setCityName] = useState('');
+  const [selectedCity, setSelectedCity] = useState('');
+  const [results, setResults] = useState([]);
+  const [chooseCityForecast, setChooseCityForecast] = useState([]);
+
+  useEffect(() => {
+    cityName.length > 3 && loadCities();
```

Comment:
1. please move 3 to separate variable (and use it everywhere you need to check number of chars)
2. `loadCities` should be defined in dependencies array (don't forget to check deps array in every hook you use)
3. just recommendation, not a rule - it would be a bit more clear to call function inside if statement

Reply:
1. Fixed.
2. Fixed.
3. Fixed.